### PR TITLE
#317 Fixing Warnings during test : test_actors.py

### PR DESCRIPTION
**Pull request by amitchaudhari9121**
Nothing serious, just the #317 warnings about test_actors.py
---
I've changed: 
- the occurrences of deprecated `np.data_type` to `datatype` in our code, and some similar changes. they won't break anything (hopefully)
- modified the `pytest.ini` to show the recurring warnings only once.

Impact: 
  Earlier I was seeing a summary of 59k something warnings...

  Screenshot 2021-03-22 at 1 57 55 AM
  ---
  Now we are seeing 65 warnings...
  
  Screenshot 2021-03-22 at 11 35 09 AM

PS: I want to work on refactoring the code so that we become compatible with vtk 9.0.0, could you please help me with the deprecated methods we have ?

### CODE REVIEW

File: fury/tests/test_material.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def test_manifest_pbr(interactive=False):
         sphere = get_sphere('symmetric362')
         shape = (11, 11, 11, sphere.vertices.shape[0])
         fid, fname = mkstemp(suffix='_odf_slicer.mmap')
-        odfs = np.memmap(fname, dtype=np.float64, mode='w+', shape=shape)
+        odfs = np.memmap(fname, dtype=float, mode='w+', shape=shape)
```
Comment (skoudoro) [inline fury/tests/test_material.py:105]: Can you revert this change?  `float` is platform-dependent. here, they are forcing float64 for whatever reason.

File: fury/tests/test_material.py [hunk 2]
```diff
@@ -504,7 +504,7 @@ def test_manifest_standard(interactive=False):
...
```
Comment (skoudoro) [inline fury/tests/test_material.py:507]: same comment as above.
Can you revert this change?  `float` is platform-dependent. here, they are forcing float64 for whatever reason.

File: fury/ui.py [hunk 3]
```diff
@@ -2059,7 +2059,7 @@ def _set_position(self, coords):
             track_position[0] += self.track.size[0] / 2.
 
         self.track.position = track_position
-        self.handle.position = self.handle.position.astype('float64')
+        self.handle.position = self.handle.position.astype('float')
```
Comment (skoudoro) [inline fury/ui.py:2062]: Can you remove the quote, they are useless. Thank you

File: fury/ui.py [hunk 4]
```diff
@@ -2415,8 +2415,8 @@ def _set_position(self, coords):
             track_position[0] -= self.track.size[0] / 2.
         self.track.position = track_position
 
-        self.handles[0].position = self.handles[0].position.astype('float64')
-        self.handles[1].position = self.handles[1].position.astype('float64')
+        self.handles[0].position = self.handles[0].position.astype('float')
```
Comment (skoudoro) [inline fury/ui.py:2418]: Can you remove the quote, they are useless. Thank you

File: fury/ui.py [hunk 5]
```diff
@@ -2415,8 +2415,8 @@ def _set_position(self, coords):
...
+        self.handles[1].position = self.handles[1].position.astype('float')
```
Comment (skoudoro) [inline fury/ui.py:2419]: Can you remove the quote, they are useless. Thank you

File: fury/utils.py [hunk 6]
```diff
@@ -94,7 +94,7 @@ def numpy_to_vtk_cells(data, is_coords=True):
         connectivity + offset information
 
     """
-    data = np.array(data)
+    data = np.array(data,dtype="object")
```
Comment (skoudoro) [inline fury/utils.py:97]: Can you remove the quote, they are useless. Thank you

File: pytest.ini [hunk 7]
```diff
@@ -1,3 +1,9 @@
...
 filterwarnings =
-    ignore: Converting `np.character` to a dtype is deprecated. :DeprecationWarning
\ No newline at end of file
+    ignore: Converting `np.character` to a dtype is deprecated. :DeprecationWarning
+    # a BIG part of 60k something warnings on pytest -svv fury/tests/test_actors.py are due to deprecated use of <np.data_type> in dipy & vtk, 
+    # where simply data_type could be used, hence ignoring the whole packages.
+    # need to refactor to something where only neglect the np.data_type deprecation warnings
+    once:::dipy
+    once:::vtk
+
```
Comment (skoudoro) [inline pytest.ini:9]: could you also manage / add this warning: 
```python
 /Users/koudoro/miniconda3/envs/fury-env-37/lib/python3.7/site-packages/vtkmodules/util/numpy_support.py:75: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    _vtk_np = {vtkConstants.VTK_BIT:numpy.bool,
```

**Review by skoudoro — COMMENTED** [review]
Hi @amitchaudhari9121, 

Thank you for doing this. See below for a quick review. Also, could you fix the pep8 issue and it will be ready to go. 

Thank you

File: pytest.ini (hunk 7 above)
Comment (amitchaudhari9121) [reply pytest.ini]: Done!
All we can do is ignore it until the devs for vtkmodules fix it by changing `np.bool` to either `bool` or `np.bool_`

We can get rid of it by changing `once` to `ignore` for those specific packages. i don't want to do that because some warnings are useful (speculating, I have not used this library enough yet :) )

File: fury/tests/test_material.py (hunk 1 above)
Comment (amitchaudhari9121) [reply fury/tests/test_material.py]: I had not thought about that!
Thanks!

File: pytest.ini (hunk 7 above)
Comment (skoudoro) [reply pytest.ini]: I agree, thanks

**Comment by amitchaudhari9121** [comment]
> Thank you for the update. It is just missing the [pep8 fix](https://github.com/fury-gl/fury/pull/396#issuecomment-803788534) and it is ready to go in.

Sorry, its done now! I somehow completely misread the warning earlier. 😅

**Comment by skoudoro** [comment]
ok, Thank you for this update. merging
